.gitlab-ci.yml

```diff
@@ -439,7 +439,6 @@ agent-bundle-windows:
 .ta-trigger:
   rules:
     - <<: *main-condition
-    - <<: *release-condition
```

Comment:
Shouldn't this be a ta-release-condition instead? This way you can add a label to trigger the TA release only with its only label as it is done for mass deployments.